features/support/env.rb

```diff
@@ -40,28 +66,28 @@ def dm_frontend_domain()
   ENV['DM_FRONTEND_DOMAIN']
 end
 
-def dm_admin_uname()
-  ENV['DM_ADMINISTRATORNAME']
+def dm_admin_email()
+  ENV['DM_ADMINISTRATORNAME'] || ENV['DM_ADMIN_EMAIL']
```

Comment:
The reason why I kept both was so that we wouldn't have failing checks between this getting merged and the jobs being changed. I won't be letting the old environment variables hang around for long.